ustriage/ustriage.py

```diff
@@ -828,16 +870,21 @@ def launch():
                   'end': args.end_date}
 
     if args.age is False and (args.show_subscribed or args.show_tagged):
-        args.age = 7
+        args.age = 6
```

Comment:
Lowering this is to reduce noise from the weekly bug scrub session I guess?

Reply:
Yes we started with the default to 7, realized that 6 is much better and since then always override to 6.